TrashMob/client-app/src/components/VolunteerOpportunities.tsx

```diff
@@ -28,24 +28,19 @@ export const VolunteerOpportunities: React.FC = () => {
         <>
             <HeroSection Title='Recruiting' Description='We’d love to have you join us.' />
             <Container>
-                <Row>
-                    <Col md={8} className='mb-2'>
+                <Row className='my-4'>
+                    <Col md={6} className='mb-2 text-center'>
                         <h1 className='m-0'>Open volunteer positions (4)</h1>
                     </Col>
                     <Col md={2} />
-                    <Col md={2} className='m-0 text-right mb-2'>
-                        <Link className='btn btn-primary ml-5 py-md-3 banner-button' to='/contactus'>
+                    <Col md={4} className='m-0 text-center mb-2 align-middle d-flex'>
```

Comment:
`align-middle` (vertical-align: middle) doesn't do anything here. Because `<Link>` is display block.

I see you add `display: flex` here, so maybe you mean `content-center` (align-content: center) ?

Reply:
Good point! Not sure what I was trying to do here, it might be a remnant of another approach I took for styling it.